slu-scripts/src/device-booting/runSendData.ts

```diff
@@ -0,0 +1,45 @@
+import * as fs from 'fs';
+import { execSync } from 'child_process';
+import 'dotenv/config';
+
+const keyFile = process.env.KEY_FILE;
+const inputEnc = process.env.INPUT_ENC;
+const isApiKey = process.env.IS_API_KEY;
+const isBaseUrl = process.env.IS_BASE_URL;
+const isAuthUrl = process.env.IS_AUTH_URL;
+const sendDataInterval = process.env.SEND_DATA_INTERVAL;
+const sensorData = process.env.SENSOR_DATA;
+const collectorBaseUrl = process.env.COLLECTOR_BASE_URL;
+const scriptsPath = process.env.SCRIPTS_PATH;
+
+export function runSendData(
+	keyFile: string | undefined,
```

Comment:
I think you should not define them as undefined. The check for undefined or empty is required but in fact they are mandatory otherwise we throw an error. So please remove `| undefined` from the parameters